draft-ietf-acme-acme.md

```diff
@@ -2193,10 +2193,11 @@ that have been provisioned to a web server.
 The server is said to "finalize" the authorization when it has completed
 one of the validations.  This is done by assigning the authorization a status of "valid"
 or "invalid", corresponding to whether it considers the account authorized
-for the identifier. If the final state is "valid", then the server MUST include
-an "expires" field. When finalizing an authorization, the server MAY remove
-challenges other than the one that was completed, and it may modify the "expires"
-field. The server SHOULD NOT remove challenges with status "invalid".
+for the identifier. If the final state is "valid", then the server MUST set
+an "expires" field, and MAY update the "expires" field if one was already present.
```

Comment:
I'm not thrilled with adding a MAY here.  I think it's clear enough that an earlier "expires" field can be overridden.